extensions/sns/README.md

```diff
@@ -9,90 +9,38 @@ dfx sns [subcommand] [flag]
+| [`init-config-file validate`](#_dfx_sns_init-config-file_validate) | Checks whether the sns config file is valid.                                                       |
+| [`deploy-testflight`](#_dfx_sns_deploy-testflight)                 | Creates a test deployment of the SNS canisters according to the local config.                      |
+| [`prepare-canisters`](#_dfx_sns_prepare-canisters)                 | Prepares dapp canister(s) for SNS decentralization by adding NNS root as one of their controllers. |
+| [`propose`](#_dfx_sns_propose)                                     | Submits a CreateServiceNervousSystem NNS Proposal.                                                 |
+| `help`                                                             | Displays usage information message for a specified subcommand.                                     |
 
 To view usage information for a specific subcommand, specify the subcommand and the `--help` flag. For example, to see usage information for `dfx sns validate`, you can run the following command:
 
 ``` bash
 dfx sns validate --help
 ```
 
-
-## dfx sns create
-
-Use the `dfx sns create` command to create an SNS configuration file. The configuration file specifies important, legally and financially relevant details about dapp decentralization.  The file leaves blank parameters such as token name; you will need to fill these in.
-
-### Basic usage
-
-``` bash
-dfx sns create
-```
-
-### Examples
-
-You can use the `dfx sns create` command to create and view a configuration file:
-
-``` bash
-dfx sns create
-less sns.yml
-```
-
-## dfx sns validate
+## dfx sns init-config-file validate
 
 Use the `dfx sns validate` command to verify that an SNS configuration file is well formed.
 
 ### Basic usage
 
 ``` bash
-dfx sns validate
-```
-
-### Examples
-
-You can use the `dfx sns validate` command to verify that a configuration template is valid.  It is not; it needs details such as token name:
-
-``` bash
-dfx sns config create
-```
-Fill in the blank fields, then:
-``` bash
-dfx sns config validate
+dfx sns init-config-file validate
 ```
 
-## dfx sns deploy
+## dfx sns deploy-testflight
 
-Use the `dfx sns deploy` command to create SNS canisters according to the local configuration file.
-
-Note:  Deploying SNS canisters does not require a proposal, however there is a hefty fee.  Please don't create canisters on mainnet until you have tested your configuration locally and are sure that you are happy with it.
+Use the `dfx sns deploy-testflight` command to create a testflight deployment of the SNS canisters according to the local configuration file. A testflight deploys an sns directly to a subnet, skipping the proposal, token swap, and sns-wasms canister. The SNS canisters remain controlled by the developer after deployment. See [the testflight documentation](https://internetcomputer.org/docs/current/developer-docs/daos/sns/testing/testing-on-mainnet) for more details.
```

Comment:
NCR

> to a subnet

This seems like a non-essential detail. How about "to a local server or the Internet Computer,"

orthogonally, s/deploys/is/

Reply:
Done!